x-pack/plugins/apm/kibana.json

```diff
@@ -12,6 +12,7 @@
     "infra"
   ],
   "optionalPlugins": [
+    "spaces",
```

Comment:
Is there any advantage in making this optional? I think we should prefer to make things required since we won't have to test with and without the plugin disabled.

Reply:
If it's disabled or unavailable for some reason APM will no longer start if it's required. If it's optional APM will still work. That's the only difference afaik